src/index.js

```diff
@@ -305,8 +310,8 @@ class Dayjs {
       HH: Utils.s(this.$H, 2, '0'),
       h: get$H(1),
       hh: get$H(2),
-      a: this.$H < 12 ? 'am' : 'pm',
-      A: this.$H < 12 ? 'AM' : 'PM',
+      a: meridiemFunc(this.$H, this.$m, true),
```

Comment:
what if you change the name `a` into `am` thats readable object or key, wdyt ?

Reply:
I think no. This lib is designed to keep the same with moment.js

Reply:
got it